### DISCUSSION TITLE
I need an Explanation here please!

### DISCUSSION THREAD

**Discussion by AlexJofer** [Chia-Network/chia-blockchain#4705] [Support]
Can someone tell me what that is?! 

![Sem Título](https://user-images.githubusercontent.com/83883461/118000884-1ebd5180-b33e-11eb-90b9-fa2de7d195f9.png)

**Comment by lvcivs** [comment]
When someone makes a transaction they can optionally add a fee. The fee will go to the farmer who farmed the block in which the transaction is included.

This is used to give transactions a priority. This comes into play if there are many pending transactions,  not all ofwhich can be included in the next block. In that case the ones with higher fees will be prioritized. As long as there are not many transactions, no fee is needed, so at the moment a fee of 0 is usually OK.

So if it was you who farmed the block in question, it means you got lucky that someone paid you a small fee of 0.0001 XCH which you receive in addition to the farmer reward of 2 XCH.

**Reply by AlexJofer** [reply]
Ok, thanks for your answer!

Best regards

**Reply by lvcivs** [reply]
You are welcome. If you are happy with this answer, you can select it as the correct answer and this topic will be marked as solved.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
